docs/codeql/reusables/supported-versions-compilers.rst

```diff
@@ -4,40 +4,41 @@
    Go (aka Golang), "Go up to 1.24", "Go 1.11 or more recent", ``.go``
-   Java,"Java 7 to 24 [5]_","javac (OpenJDK and Oracle JDK),
+   Java,"Java 7 to 24 [6]_","javac (OpenJDK and Oracle JDK),
 
-   Eclipse compiler for Java (ECJ) [6]_",``.java``
+   Eclipse compiler for Java (ECJ) [7]_",``.java``
    Kotlin,"Kotlin 1.5.0 to 2.1.2\ *x*","kotlinc",``.kt``
-   JavaScript,ECMAScript 2022 or lower,Not applicable,"``.js``, ``.jsx``, ``.mjs``, ``.es``, ``.es6``, ``.htm``, ``.html``, ``.xhtm``, ``.xhtml``, ``.vue``, ``.hbs``, ``.ejs``, ``.njk``, ``.json``, ``.yaml``, ``.yml``, ``.raml``, ``.xml`` [7]_"
-   Python [8]_,"2.7, 3.5, 3.6, 3.7, 3.8, 3.9, 3.10, 3.11, 3.12, 3.13",Not applicable,``.py``
-   Ruby [9]_,"up to 3.3",Not applicable,"``.rb``, ``.erb``, ``.gemspec``, ``Gemfile``"
-   Swift [10]_,"Swift 5.4-6.0","Swift compiler","``.swift``"
-   TypeScript [11]_,"2.6-5.8",Standard TypeScript compiler,"``.ts``, ``.tsx``, ``.mts``, ``.cts``"
+   JavaScript,ECMAScript 2022 or lower,Not applicable,"``.js``, ``.jsx``, ``.mjs``, ``.es``, ``.es6``, ``.htm``, ``.html``, ``.xhtm``, ``.xhtml``, ``.vue``, ``.hbs``, ``.ejs``, ``.njk``, ``.json``, ``.yaml``, ``.yml``, ``.raml``, ``.xml`` [8]_"
+   Python [9]_,"2.7, 3.5, 3.6, 3.7, 3.8, 3.9, 3.10, 3.11, 3.12, 3.13",Not applicable,``.py``
+   Ruby [10]_,"up to 3.3",Not applicable,"``.rb``, ``.erb``, ``.gemspec``, ``Gemfile``"
+   Swift [11]_,"Swift 5.4-6.0","Swift compiler","``.swift``"
+   TypeScript [12]_,"2.6-5.8",Standard TypeScript compiler,"``.ts``, ``.tsx``, ``.mts``, ``.cts``"
 
 .. container:: footnote-group
 
     .. [1] C++20 modules are *not* supported.
-    .. [2] Objective-C, Objective-C++, C++/CLI, and C++/CX are not supported.
-    .. [3] Support for the clang-cl compiler is preliminary.
-    .. [4] Support for the Arm Compiler (armcc) is preliminary.
-    .. [5] Builds that execute on Java 7 to 24 can be analyzed. The analysis understands standard language features in Java 8 to 24; "preview" and "incubator" features are not supported. Source code using Java language versions older than Java 8 are analyzed as Java 8 code.
-    .. [6] ECJ is supported when the build invokes it via the Maven Compiler plugin or the Takari Lifecycle plugin.
-    .. [7] JSX and Flow code, YAML, JSON, HTML, and XML files may also be analyzed with JavaScript files.
-    .. [8] The extractor requires Python 3 to run. To analyze Python 2.7 you should install both versions of Python.
-    .. [9] Requires glibc 2.17.
-    .. [10] Support for the analysis of Swift requires macOS.
-    .. [11] TypeScript analysis is performed by running the JavaScript extractor with TypeScript enabled. This is the default.
+    .. [2] C23 and C++23 support is currently in beta.
+    .. [3] Objective-C, Objective-C++, C++/CLI, and C++/CX are not supported.
+    .. [4] Support for the clang-cl compiler is preliminary.
```

Comment:
What does `preliminary` support mean?

Reply:
I don't know. Note that this was already there. I just updated the number on that line to make the footnotes work properly.

Reply:
From a C/C++ point of view, do we support those compilers?